hieradata/common.yaml

```diff
@@ -314,6 +314,15 @@ govuk::apps::publisher::redis_port: "%{hiera('sidekiq_port')}"
 govuk::apps::short_url_manager::redis_host: "%{hiera('sidekiq_host')}"
 govuk::apps::short_url_manager::redis_port: "%{hiera('sidekiq_port')}"
 
+govuk::apps::whitehall::admin_db_hostname: whitehall-master.mysql
+govuk::apps::whitehall::admin_key_space_limit: '262144'
+govuk::apps::whitehall::admin_db_name: whitehall_production
+govuk::apps::whitehall::admin_db_password: "%{hiera('govuk::apps::whitehall::db::mysql_whitehall_admin')}"
+govuk::apps::whitehall::admin_db_username: whitehall
+govuk::apps::whitehall::db_hostname: whitehall-slave.mysql
+govuk::apps::whitehall::db_name: whitehall_production
+govuk::apps::whitehall::db_password: "%{hiera('govuk::apps::whitehall::db::mysql_whitehall')}"
+govuk::apps::whitehall::db_username: whitehall_fe
```

Comment:
I think it's right looking at https://github.gds/gds/alphagov-deployment/blob/master/whitehall/to_upload/database.yml#L5 vs. https://github.gds/gds/alphagov-deployment/blob/master/whitehall/admin_to_upload/config/database.yml.erb#L5

The former is the username for the frontend configuration of the app, the latter is the backend one.